doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -0,0 +1,398 @@
+.. _autopts-linux:
+
+AutoPTS on Linux (Ubuntu)
+##########################
+
+Overview
+========
+
+This tutorial shows how to setup AutoPTS client on Linux with AutoPTS server running on Windows 10
+virtual machine. Tested with Ubuntu 20.4 and Linux Mint 20.4.
+
+Setup Linux
+===========================
+
+
+
+Setup Zephyr project
+---------------------
+
+Do the setup from Zephyr site https://docs.zephyrproject.org/latest/getting_started/index.html,
+especially:
+
+Update OS
+^^^^^^^^^^
+
+This guide covers Ubuntu version 18.04 LTS and later.
+
+.. code-block::
+
+    sudo apt update
+    sudo apt upgrade
+
+Install dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block::
+
+    sudo apt install --no-install-recommends git cmake ninja-build gperf \
+    ccache dfu-util device-tree-compiler wget \
+    python3-dev python3-pip python3-setuptools python3-tk python3-wheel xz-utils file \
+    make gcc gcc-multilib g++-multilib libsdl2-dev
+
+Get Zephyr and install Python dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Install west, and make sure ~/.local/bin is on your PATH environment variable:
+
+.. code-block::
+
+    pip3 install --user -U west
+    echo 'export PATH=~/.local/bin:"$PATH"' >> ~/.bashrc
+    source ~/.bashrc
+
+Get the Zephyr source code:
+
+.. code-block::
+
+    west init ~/zephyrproject
+    cd ~/zephyrproject
+    west update
+
+Export a Zephyr CMake package. This allows CMake to automatically load
+boilerplate code required for building Zephyr applications:
+
+.. code-block::
+
+    west zephyr-export
+
+Zephyr’s scripts/requirements.txt file declares additional Python dependencies.
+Install them with pip3:
+
+.. code-block::
+
+    pip3 install --user -r ~/zephyrproject/zephyr/scripts/requirements.txt
+
+Install a Toolchain
+^^^^^^^^^^^^^^^^^^^^
+
+A toolchain provides a compiler, assembler, linker, and other programs
+required to build Zephyr applications.
+
+Download the latest SDK installer:
+
+.. code-block::
+
+    cd ~
+    wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.12.4/zephyr-sdk-0.12.4-x86_64-linux-setup.run
```

Comment:
One of my concerns with this is that you will need to update the SDK version here and in some other places every time a new version comes out, so there's a risk of out-dated references. Would it make sense to instead just link to the SDK installation page?